README.md

```diff
@@ -12,6 +12,9 @@ one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,
 
 With this formal definition, we can argue with it to prove that these two graphs are isomorphic. Within each graph, they have the same number of nodes, no matter if it is A or B. They are also completely connected, hence for every vertex there is an edge going to another vertex. Each graph contains this. If we wanted to map them in another way, we still have the same number of nodes, so the same number of vertices crossing over. For example, if we have two nodes from graph A and try to map those to all the nodes in graph B if graph B had 10 nodes, this would not be a bijection. The reason for this is because they do not have a one-to-one mapping, resulting in these two graphs not being isomorphic. In this problem however, since graph A and graph B have the same number of nodes, mapping each node to the other will allow for pairs of nodes to be connected not leaving any out and no nodes being mapped multiple times. Hence, these two graphs are isomorphic. Looking into the graph lecture, we can find if two nodes are connected by the degree. For instance if two nodes contain an edge, are connected, we will see an e with a subdegree showing they are connected. If both graphs contain the same degrees through the same nodes that are connected they are isomorphic. So, like in the slides, we can ignore vertex names and if they have the same node pairings as descriped in the above e subdegree statement, they have the same structure. 
 
+Revision in response to latest change request: 
+To map nodes between graphs, we use the degree of each node, as shown in the lecture. Starting with a node in graph $A$, we check its degree(the number of edges connected to it). In graph $B$, we find a node with the same degree and map them. Repeating this process for all nodes, making sure each node in $A$ is paired with one in $B$ that has the same degree. For example, if a node in A has a degree of 1(connected to one other node by a single edge), we find a node in $B$ with a degree of 1 and map them(as shown in the lecture video). This process continues until all nodes are paired.
```

Comment:
But all nodes have the same degree. And please revise your text, don't add new things that changes what you said earlier at the end.